### PR TITLE
Added alertmanager for prometheus to send email alerts

### PR DESCRIPTION
**Pull request by bamachrn**
This PR is for configuring the alert managers. These alertmangers are for alerting the target audiances from prometheus.
It adds the following:
* Alertmanager: alertmanager-appstudio
* Service for exposing alertmanagers: alertmanager-appstudio
* AlertmanagerConfig: config-appstudio-stage

Signed-off-by: Bama Charan Kundu <bamachrn@gmail.com>

### CODE REVIEW

**Comment by bamachrn** [comment]
@gabemontero @amfred we need smtp whitelisting and configurations to be able to send alerts from prometheus.

File: components/monitoring/prometheus/base/alermanager-config.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+apiVersion: monitoring.coreos.com/v1
+kind: Alertmanager
+metadata:
+  name: alertmanager-appstudio
```
Comment (amfred) [inline components/monitoring/prometheus/base/alermanager-config.yaml:4]: Let's use alertmanager-stonesoup

File: components/monitoring/prometheus/base/alermanager-config.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  namespace: appstudio-workload-monitoring
```
Comment (amfred) [inline components/monitoring/prometheus/base/alermanager-config.yaml:5]: Let's leave this alone for now

File: components/monitoring/prometheus/base/alermanager-config.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+spec:
+  replicas: 3
+  alertmanagerConfigSelector:
+    matchLabels:
+      cluster: appstudio-stage
```
Comment (amfred) [inline components/monitoring/prometheus/base/alermanager-config.yaml:10]: Cluster names are at the top of the Stonesoup Cluster Deployment Notes doc.

Comment (amfred) [reply components/monitoring/prometheus/base/alermanager-config.yaml:10]: Per our discussion, let's get the cluster name from a variable.

**Comment by gbenhaim** [comment]
@bamachrn we are going to use Alert Manager provided by RHOBS. I think that we can close this PR.

**Comment by bamachrn** [comment]
not required anymore

**Pull request closed without merging**